charts/pega/templates/_pega-service.tpl

```diff
@@ -25,6 +25,7 @@ spec:
   - name: http
     port: {{ .port }}
     targetPort: {{ .targetPort }}
+  sessionAffinity: ClientIP
```

Comment:
This would set sessionAffinity to ClientIP for everyone.  If you are using a load balancer, how does this work along side cookie based session affinity?

The default timeoutSeconds of ~3hrs would also need to be set based on the passivation timeout like is done for the ALB.